components/FullEventCard/FullEventCard.js

```diff
@@ -54,8 +55,8 @@ export default function FullEventCard({ route }) {
       <Text style={styles.label}>EQUIPMENT:</Text>
 
         <View>
-          {event.equipment.map(e => {
-            return <Text style={styles.equipmentList}>• {e.name}</Text>
+          {event.equipment.split(',').map(e => {
+            return <Text style={styles.equipmentList}>• {e}</Text>
```

Comment:
I think this will work well to print equipment as bullet points. Hopefully, we can reach our stretch goal of adding the ability for a user to interact with equipment lists.